keras_nlp/models/xlm_roberta/xlm_roberta_tokenizer.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, proto, **kwargs):
         self.pad_token_id = 1  # <pad>
         self.end_token_id = 2  # </s>
         self.unk_token_id = 3  # <unk>
+        self.mask_token_id = 4
```

Comment:
here adding the mask_token_id and assigning 4 to it, will make a conflict with the sentencepiece model you are using in your tests because the re-mapping of XLMRoberta tokenizer

thanks for @abheesht17 to clarify the re-mapping here:
https://github.com/keras-team/keras-nlp/issues/930#issuecomment-1486280061

so first thing to preserve compatibility with the original implementation is to assign the mask_token_id to the last index (vocab size + 1) . the one is because XLMRoberta tokenizer will always discard the first three tokens in the vocab and add the vocabulary_prefix. 
after adding mask_toke_id and also `_mask_token =<mask>` handle it also in the first 4 functions in the class.